proj/src/main/webapp/chat.css

```diff
@@ -1,5 +1,12 @@
 @import url('https://fonts.googleapis.com/icon?family=Material+Icons');
 
+/* styling for hiding something */
+.hidden {
+    position: absolute;
+    left: -999px;
+    bottom: -999px;
```

Comment:
why not use `display:hidden` or `display:none`?

Reply:
i tried using display hidden, display none, and .hidden = true but none of them worked for some reason